infra/compile.py

```diff
@@ -661,15 +708,25 @@ def compile(tree, ctx, indent=0):
             out += "\n" + " " * indent + "constants.{} = {}_constants.{};".format(const, tree.module, const)
         return out
     elif isinstance(tree, ast.ClassDef):
-        assert not tree.bases
-        assert not tree.keywords
-        assert not tree.decorator_list
+        if tree.decorator_list:
+            assert tree.decorator_list[0].func.value.id == "wbetools"
```

Comment:
I'm a little worried about the lack of an `isinstance` assertion on the `tree.decorator_list[0].func`, and also the fact that it's not testing every decorator (just the first) though I'm sure it's fine.